src/ModelFactory.h

```diff
@@ -79,6 +80,7 @@ class ModelFactory
 {
 public:
     static std::unique_ptr<ModelConfig> Create(const std::string& name, const SNR&);
```

Comment:
Create( model, snr ) - but we can't because of the way MonoMolecularIntegrator and CCS are currently structured.   I'd still like to add the Create( read ) API however, as that helps me simplify some code in LAA.